src/pages/entity/repository/history/[id].tsx

```diff
@@ -237,6 +240,21 @@ const query = gql`
           }
         }
       }
+      ... on Solution {
+        currentRevision {
+          id
+        }
+        solutionRevisions: revisions {
```

Comment:
I needed to add a label, because otherwise, I get an error:

![grafik](https://user-images.githubusercontent.com/13507950/118184964-48f53900-b43c-11eb-9e2e-3e42d7acd7e3.png)

@kulla:Do you have an idea why this is only happening for revisions of solutions?